Sources/Sentry/SentrySessionTaskSearch.m

```diff
@@ -0,0 +1,57 @@
+/**
+ * Part of this code was copied from
+ * https://github.com/AFNetworking/AFNetworking/blob/4eaec5b586ddd897ebeda896e332a62a9fdab818/AFNetworking/AFURLSessionManager.m#L407-L418
```

Comment:
```suggestion
 * https://github.com/AFNetworking/AFNetworking/blob/4eaec5b586ddd897ebeda896e332a62a9fdab818/AFNetworking/AFURLSessionManager.m#L349-L418
```

Reply:
@brustolin, is mentioning AFNetworking like this fine for the MIT license?

Reply:
yeah LGTM. Possibly throw a link to their license too right below it but not required since we're both on MIT and the code copied is clear here
```suggestion
 * https://github.com/AFNetworking/AFNetworking/blob/4eaec5b586ddd897ebeda896e332a62a9fdab818/AFNetworking/AFURLSessionManager.m#L349-L418
https://github.com/AFNetworking/AFNetworking/blob/4eaec5b586ddd897ebeda896e332a62a9fdab818/LICENSE
```